### PR TITLE
Updated bundle to use strict standard 2.0.

### LINKED ISSUE #29: Upgrade codesniffer standard to 2.0

### PR DESCRIPTION
**Pull request by emaslak**
Closes #29

### CODE REVIEW

**Comment by emaslak** [comment]
Should be merged after strict standard update, since there are some bugs right now in version 2.1.

**Comment by saimaz** [comment]
Please make sure tests are passing.

File: .travis.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ before_script:
   - beanstalkd -d -p 127.0.0.1 -p 11300
 script:
   - vendor/bin/phpunit --coverage-text --coverage-clover=coverage.clover
-  - vendor/bin/phpcs -p --standard=$TRAVIS_BUILD_DIR/vendor/ongr/ongr-strict-standard/ONGR --ignore=vendor,Tests/app ./
+  - vendor/bin/phpcs -p --standard=$TRAVIS_BUILD_DIR/vendor/ongr/ongr-strict-standard/Ongr --ignore=vendor,Tests/app ./
```
Comment (saimaz) [inline .travis.yml]: Remove travis build dir

File: composer.json [hunk 2]
```diff
@@ -23,12 +23,12 @@
     "require-dev": {
         "mikey179/vfsStream": "~1.4",
         "phpunit/phpunit": "~4.1",
-        "squizlabs/php_codesniffer": "~1.5",
-        "ongr/ongr-strict-standard": "~1.0.0-beta",
+        "squizlabs/php_codesniffer": "~2.0",
```
Comment (saimaz) [inline composer.json]: remove code sniffer

**Comment by saimaz** [comment]
In terms of segmentation fault leave it, we will fix it.

**Pull request closed without merging**